examples/get-started/pure-js-without-map/package.json

```diff
@@ -8,11 +8,10 @@
     "build": "NODE_ENV=production webpack --env.prod=true"
   },
   "dependencies": {
-    "@deck.gl/core": "^5.3.0-rc",
-    "@deck.gl/layers": "^5.3.0-rc"
+    "@deck.gl/core": "^6.0.0-beta",
```

Comment:
not `^6.0.0-beta.1` ?

Reply:
`^6.0.0-beta` works. See https://semver.npmjs.com/

Reply:
👍